local_development.yml

```diff
@@ -0,0 +1,274 @@
+      when: ansible_distribution != "MacOSX"
+      block:
+        - name: Install commands via package manager
+          become: yes
+          ansible.builtin.package:
+            name:
+              - pip
+              - git
+              - terraform
+              - kubectl
+              - google-cloud-sdk
+              - azure-cli
+            state: present
+        
+        - name: Install commands via download 
+          block:
+            # Determine if the executable already exists in path
+            - name: Check if commands are already available
+              ansible.builtin.command:
+                cmd: "which {{ item.exe }}"
+              register: command_availability
+              ignore_errors: true
+              loop:
+                - exe: "aws-iam-authenticator"
+                  download_url: "https://amazon-eks.s3.us-west-2.amazonaws.com/1.21.2/2021-07-05/bin/linux/amd64/aws-iam-authenticator"
+
+            # If command found not to be available download and install
+            - name: Download and install any required commands (may take some time)
+              become: yes
+              ansible.builtin.get_url:
+                url: "{{ item.item.download_url }}"
+                dest: "/usr/local/bin/{{ item.item.exe }}"
+                mode: 0555
+              when: item.rc != 0
+              loop: "{{ command_availability.results }}"
+
+    - name: Ensure availability of system commands (MacOSX)
+      when: ansible_distribution == "MacOSX"
+      community.general.homebrew:
+        name:
+          - git
+          - terraform
+          - kubectl
+          - aws-iam-authenticator
+          #- azure-cli
+        state: present
+      tags: system
+
+    - name: Ensure availability of Gcloud system commands (MacOSX)
+      when: ansible_distribution == "MacOSX"
+      community.general.homebrew:
+        name: google-cloud-sdk
+        state: present
+      register: __osx_gcloud
+      failed_when: not (__osx_gcloud.msg is search("already installed") or __osx_gcloud.msg is search("Update done"))
+      tags: system
+
+    # TODO Add azure-mgmt-netapp to python_azure.txt
+    - name: Set up the Python dependencies
+      ansible.builtin.pip:
+        requirements: "{{ item }}"
+        chdir: "./payload/deps"
+        state: present
+      loop:
+        - python_base.txt
+        - python_secondary.txt
+        - python_aws.txt
+        - python_gcp.txt
+        - python_azure.txt
+      tags: python
+
+    - name: Install the CDP CLI itself
+      ansible.builtin.pip:
+        name: cdpcli-beta
+        state: present
+      tags: python
+
+    - name: Clone the Cloudera automation collections
+      ansible.builtin.git:
+        repo: "{{ item.repo }}"
+        dest: "{{ lookup('env', 'ANSIBLE_COLLECTIONS_PATH') }}/ansible_collections/cloudera/{{ item.collection}}"
+      loop:
+        - repo: git@github.com:cloudera-labs/cloudera.cloud.git
```

Comment:
Do you want to clone these as devel?

Reply:
No, I want to leave that to the user. I will add a note/callout to highlight this fact.